src/buttons/dailyInterval.js

```diff
@@ -10,7 +10,7 @@ const modalObject = {
           type: 4,
           style: 1,
           custom_id: "input",
-          label: "Enter a 24 hour dailymsg interval (HH:MM).",
+          label: "At what time should the daily message be posted? (HH:MM)",
```

Comment:
```suggestion
          label: "When should the daily message be posted? (HH:MM)",
```
Perhaps this could be better?

Reply:
Which buttons are you referring to?

Reply:
> Which buttons are you referring to?

For example the buttons in the settings menu that open this modal.

Reply:
So you want to rename the "daily interval" completely? What do you want to call it?

My approach would be "time"/"daily messages time".

Also do you want to write "message" or "messages"? (Shouldn't it be singular?)

Reply:
Thats a good question maybe "Post time" works.

Reply:
> Also do you want to write "message" or "messages"? (Shouldn't it be singular?)

Any thoughts on this one?

Reply:
Can be singular as its only one message yes